app/components/agenda/agenda-overview/agenda-overview-item.hbs

```diff
@@ -205,7 +205,16 @@
         <div class="vlc-agenda-items__remarks">
           {{!-- Empty container below so that, when it is empty, the adjacent element is still pushed to the right  --}}
           <div>
-            {{#if @agendaitem.privateComment}}
```

Comment:
I might add a permission check here for good measure.
But if I use "manage-agendaitems" then OVRB and KB will no longer be able to see this comment.
They can see it now on prod

Reply:
If there's no check on PROD now, I wouldn't add it. Could lead to an unintended side-effect

Reply:
I've checked other profiles. A non-editor profile on this view does not have the model for internalReviews (as expected, yggdrasil needs to be told explicitly what to collect.
OVRB and KB still see the comment as they did before, but only in the agenda views, not subcase view